modules/planning/tasks/deciders/speed_bounds_decider/st_boundary_mapper.cc

```diff
@@ -305,7 +313,7 @@ bool STBoundaryMapper::GetOverlapBoundaryPoints(
               const auto& point_high = discretized_path.Evaluate(
                   high_s + discretized_path.front().s());
               if (!CheckOverlap(point_high, obs_box,
-                                FLAGS_nonstatic_obstacle_nudge_l_buffer)) {
+                                l_buffer)) {
```

Comment:
Try to refine the layout after making change.

Reply:
Will fix. Thank you.